app/Controller/CoGroupsController.php

```diff
@@ -753,7 +753,9 @@ public function paginationConditions() {
     // First get the CoPersonID to work against either the current user, or the user being acted on in select mode
     $coPersonId = $this->Session->read('Auth.User.co_person_id');
     if($this->action == 'select') {
```

Comment:
For simplicity, might we just add the test for a non-empty coPersonId here?

Reply:
Done